saleor/app/manifest_validations.py

```diff
@@ -178,31 +180,43 @@ def clean_extensions(manifest_data, app_permissions, errors):
         _clean_extension_permissions(extension, app_permissions, errors)
 
 
+def clean_webhooks(manifest_data, app_permissions, errors):
+    webhooks = manifest_data.get("webhooks", [])
+    for webhook in webhooks:
+        validate_query(webhook["query"])
```

Comment:
You don't use `errors` here. 
IMO, you should iterate over a webhooks and gather all errors that you got. Currently, it will raise an exception after discovering first incorrect query.